back-end/API/app/repositories/user_repo.py

```
@@ -0,0 +1,65 @@
+    def update(self, dto: UserUpdateDTO) -> UserReadDTO:
+        user = User.query.filter_by(id=dto.id).first()
+        if not user:
+            raise NotFoundException("user not found", dto.id)
+        user.verified = dto.verified
+        user.enterpriseId = dto.enterpriseId
+        user.isModerator = dto.isModerator
+        user.active = dto.active
+        user.email = dto.email
+        user.firstName = dto.firstName
+        user.lastName = dto.lastName
+        if dto.password != None or dto.password != "":
```

Ruff (0.15.13)

[error] Comparison to `None` should be `cond is not None`

Replace with `cond is not None`

(E711)